Makefile

```
@@ -7,6 +7,26 @@ ifneq (,$(DAPPER_HOST_ARCH))
 
 include $(SHIPYARD_DIR)/Makefile.inc
 
+# Generated files
+
+GO ?= go
+MOCKGEN := $(shell $(GO) env GOPATH)/bin/mockgen
+MOCKGEN_VERSION := v1.6.0
+
+$(MOCKGEN):
+	go install github.com/golang/mock/mockgen@$(MOCKGEN_VERSION)
```

```suggestion
	$(GO) install github.com/golang/mock/mockgen@$(MOCKGEN_VERSION)
```

Since you bothered to defined $GO, wouldn't you want to use it here as well?

---

Duh, yes, thanks!